### PR TITLE
Commenting feature for application

### PR DESCRIPTION
**Pull request by johndbritton**
This is just the beginning, it needs some more work.

The goal is to add a commenting feature to the application.

### CODE REVIEW

**Comment by dice** [comment]
This looks like a great plan! :+1:

**Comment by zakuro9715** [comment]
LGTM :+1:

File: feature [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class NewFeature < OldFeature
+
+ def initialize
+   puts "hello, japan"
```
Comment (johndbritton) [inline feature]: This should be capitalized.

Comment (dice) [inline feature]: You mean,"Hello, Japan?"

File: feature [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class NewFeature < OldFeature
+
+ def initialize
```
Comment (zakuro9715) [inline feature]: Indent shoud be two spaces.

**Pull request merged**